src/test/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReaderSpec.scala

```diff
@@ -21,11 +23,54 @@ class OmnibusDaDatasetReaderSpec extends FlatSpecLike with Matchers {
   val questionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
     "make electricity and heat buildings. Explain how long it would most likely take for new coal " +
     "to form."
+  val transformedQuestionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
+    "make electricity and heat buildings. How long it would most likely take for new coal to form."
   val answerText2 = "Coal may slowly form over millions of years."
   val row2 = s""""1",,"false","${questionText2}","${answerText2}",""" +
     """"1","false","false","false","Source2","5","2015","","Tag2",""" +
     """"legacyId2","importedId2""""
 
+  val questionText3 = "Read the given statement and determine if the behavior is "+
+    "either learned or innate. A bear eating salmon caught from a river."
+  val transformedQuestionText3 = "Read the given statement and determine if the "+
+    "behavior is either learned or innate. A bear eating salmon caught from a river."
+
+  val questionText4 = "Even though an adult amphibian can live on land, why must it " +
+    "return to the water? (Explain.)"
+  val transformedQuestionText4 = "Even though an adult amphibian can live on land, " +
+    "why must it return to the water? (Explain.)"
+
+  // TODO(nelson): Would like to be able to handle plural/singular.
+  val questionText5 = "Name four organisms you might find living in or around a " +
+    "freshwater lake."
+  val transformedQuestionText5 = "What is four organisms you might find living in or around a " +
```

Comment:
I would ask Sumithra what she did about questions like this that ask for a specific number of things.